utils/status_module.conf

```diff
@@ -0,0 +1,29 @@
+# vim:set ft=tmux:
+
+# Puts a space in between modules only if
+# @catppuccin_status_connect_separator != "yes"
+set -gqF @_ctp_connect "#{?#{==:#{@catppuccin_status_connect_separator},yes},, }"
+
+# Add the left separator. Background is none (switched off).
+set -gF "@catppuccin_status_${MODULE_NAME}" \
+  "#[fg=#{@catppuccin_${MODULE_NAME}_color},bg=default,nobold,nounderscore,noitalics]#{@_ctp_connect}#{@catppuccin_status_left_separator}"
+
+# Add the icon. Background is the color, and foreground is crust.
+set -agF "@catppuccin_status_${MODULE_NAME}" \
+    "#[fg=#{@thm_crust},bg=#{@catppuccin_${MODULE_NAME}_color}]#{@catppuccin_${MODULE_NAME}_icon} "
```

Comment:
Space is not removed if icon is empty, see #265.

Reply:
Has this been fixed? Can't remember